packages/devextreme-react/tsconfig.esm.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Why do we need a separate tsconfig for esm?

Reply:
in default tsconfig. we compile to cjs, in tsconfig.esm we force compile to build esm in it, also disable generating d.ts for /esm folder